lnbits/decorators.py

```diff
@@ -306,9 +306,9 @@ async def _get_account_from_token(access_token):
     try:
         payload = jwt.decode(access_token, settings.auth_secret_key, "HS256")
         if "sub" in payload and payload.get("sub"):
-            return await get_account_by_username(payload.get("sub"))
+            return await get_account_by_username(str(payload.get("sub")))
```

Comment:
"sub" in a jwt token is usually the user-id and not a username

Reply:
Yes, I'm still in doubt here due to this scenario:
 1. `user_id` (`usr`) can be used as login method (current state, but for backwards compatibility it will still be allowed)
 1. sometimes `access-tokens` are shared, lost, stolen, etc. That is why an access token has an `expire` value (becomes invalid after a while and a new one must be obtained)

If the `user_id` is present in the token and **both** `1.` and `2.` are true then it is not possible to revoke that token anymore.